ws-nextjs-app/pages/[service]/new_live/[id]/Header/styles.tsx

```diff
@@ -42,17 +42,17 @@ export default {
     css({
       color: '#00ccc7',
```

Comment:
does this colour need to be added to the palette?

Reply:
Yes it should, this is a new colour called LIVE_LIGHT. More [here](https://github.com/bbc/simorgh/pull/10829)